R/last_date_month.R

```diff
@@ -0,0 +1,16 @@
+#' Return the end date of the month of the given date
+#'
+#' @description Return the end date of the month of the given date
+#'
+#' @param x a date with a date format
+#'
+#' @return a vector of dates of the end date of the FY year
+#' @export
+#'
+#' @examples
+#' last_date_month(lubridate::as_date("2020-02-05"))
+#'
+#' @family date functions
+last_date_month <- function(x) {
+  return(lubridate::ceiling_date(x, "month") - lubridate::days(1))
+}
```

Comment:
Is there an equivalently existing function?

Reply:
I don't think so - This looks good though, I can't think of a better / faster way to do it!